src/gt4py/backend/dace_stencil_object.py

```diff
@@ -249,8 +108,15 @@ def freeze(
         except FileNotFoundError:
             # otherwise, wrap and save sdfg from scratch
             inner_sdfg = self.sdfg()
-
-            frozen_sdfg = self._sdfg_freeze_domain_and_origin(inner_sdfg, domain, origin)
+            from gt4py.backend.dace_backend import freeze_origin_domain_sdfg
```

Comment:
Just curious: why is this here and not at the top of the file?

Reply:
I am sometimes lazy when writing code. Came back like a boomerang.